addons/common/fnc_isAttenuated.sqf

```diff
@@ -41,11 +41,13 @@ if (_turnedOut) exitWith {false};
 // open vehicle
 private _vehicle = vehicle _unit;
 private _config = configFile >> "CfgVehicles" >> typeOf _vehicle;
+
+// class CfgSoundEffects >> class AttenuationsEffects
 private _attenuationType = getText (_config >> "attenuationEffectType");
-if (_attenuationType in ["OpenCarAttenuation", "OpenHeliAttenuation"]) exitWith {false};
+if (_attenuationType in ["OpenCarAttenuation", "OpenHeliAttenuation", "jsrs_OpenCar_Attenuation", "jsrs_SemiOpenCar_Attenuation"]) exitWith {false};
```

Comment:
Can we avoid mod specific "compatibilities" in CBA?

Reply:
If you check out those classes, you can see that they don't have anything to do with whether you should or shouldn't hear anyone outside the vehicle anyway.
Tbh, it seems to me like this is something the radio mods should worry about. There is no link  from the base game to this. The ingame sounds clearly follow different rules that wouldn't work for the radio mods.

Reply:
@commy2 This line is THE key line of this function. Because of this value i get quads, boots, littlebirds and open jeeps as open vehicles. It is not important what these classes define in cfgSoundEffects... it is important that these classes give a hint that the vehicle is open to the outside. If i delete this check... the function is useless.

 @TheMagnetar yes you are right, BUT then the function is broken if someone use the clientside mod JSRS... what should i do?

Reply:
>This line is THE key line of this function.

But it isn't. It is an arbitrary list that would have to be expanded every time someone makes a new class that would qualify in a random mod. I am not going to approve and merge references to i.e. JSRS in CBA.

Reply:
>Ideally have a "registerAttenuationClass" function which adds those to a global array. Similar to how "registerFeatureCamera" is but without code stuff.

No, because vehicle attenuation as it appears in Arma is not linked to whether you should be able to hear someone outside the vehicle from inside or not.